src/preprocessing/passes/symmetry_breaker.cpp

```diff
@@ -101,23 +104,29 @@ Node SymmetryBreaker::generateSymBkConstraints(const vector<vector<Node>>& parts
   {
     return constraints[0];
   }
-  return nm->mkNode(kind::AND, constraints);
+  return nm->mkNode(AND, constraints);
 }
 
 Kind SymmetryBreaker::getOrderKind(Node node)
 {
-  if (node.getType().isInteger() || node.getType().isReal())
+  TypeNode tn = node.getType();
+  if (tn.isBoolean())
   {
-    return kind::LEQ;
+    return IMPLIES;
   }
-  else if (node.getType().isBitVector())
+  else if (tn.isReal())
   {
-    return kind::BITVECTOR_ULE;
+    return LEQ;
   }
-  else
+  else if (tn.isBitVector())
   {
-    return kind::EQUAL;
+    return BITVECTOR_ULE;
   }
+  if (tn.isFirstClass())
```

Comment:
maybe use `else` as in the above examples to be consistent? Or you could remove the `else` above

Reply:
This isn't mutually exclusive with the others, this is a type class whereas the others are types.